### DISCUSSION TITLE
How can I turn off the post display on the login page?

### DISCUSSION THREAD

**Discussion by MegaDom** [misskey-dev/misskey#14774] [Q&A]
### Summary

In "Invite Only" mode, I think non-logged in visitors cannot see the timeline without logging in.

### Purpose

 I hope that users who are not logged in cannot see any post content
Is there any other way to turn off the "Have a look at the timeline" module?

### Do you want to implement this feature yourself?

- [ ] Yes, I will implement this by myself and send a pull request

**Comment by anatawa12** [comment]
If the instance disabled local users in role policy, "have a look at the timeline" part will not be visible to users.
On the other hand, there is no way to disable the featured notes on the right side of the visitor page.
